api/src/main/java/io/opentelemetry/correlationcontext/DefaultCorrelationContextManagerProvider.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.correlationcontext;
+
+import io.opentelemetry.correlationcontext.spi.CorrelationContextManagerProvider;
+
+public final class DefaultCorrelationContextManagerProvider
```

Comment:
looks like this snuck into this PR. Can it be moved to another PR, since it doesn't seem to be directly related to it?

Reply:
Lazy to update this now, will get updated once https://github.com/open-telemetry/opentelemetry-java/pull/960 is merged, so I will wait to merge that one first.

Reply:
aha, cool.  👍

Reply:
done :)